addon/components/frost-select-li.js

```diff
@@ -24,7 +24,6 @@ export default Component.extend({
     // options
     data: PropTypes.object.isRequired,
     filter: PropTypes.string,
-    hook: PropTypes.string.isRequired,
```

Comment:
This isn't necessary because it is provided in the `frost-component` that this component extends.